src/Microsoft.Health.Fhir.Core/Features/Operations/ConvertData/ConvertDataEngine.cs

```diff
@@ -95,15 +95,7 @@ private string GetConvertDataResult(ConvertDataRequest convertRequest, ITemplate
                     throw new ConvertDataTimeoutException(Core.Resources.ConvertDataOperationTimeout, convertException.InnerException);
                 }
 
-                if (convertException is PostprocessException)
-                {
-                    _logger.LogInformation($"An exception of type {convertException.GetType()} occurred during postprocessing. {convertException.StackTrace}", "Convert data failed.");
-                }
-                else
-                {
-                    _logger.LogInformation(convertException, "Convert data failed.");
-                }
-
+                _logger.LogError($"Convert data failed. An exception of type {convertException.GetType()} occurred with error code - {convertException.FhirConverterErrorCode}. {convertException.StackTrace}");
```

Comment:
One thing we are losing is the an explicit message that the error occurred during post processing.  We should be able to infer that from the stack trace but it may still be nice to include a post processing indicator in the log for clarity.

Reply:
The type already has that information.
e.g. of how it was logged - An exception of type Microsoft.Health.Fhir.Liquid.Converter.Exceptions.PostprocessException occurred during postprocessing. 

Yeah if we have derived classes for PostProcessException later, then yes this would make sense, but that is also why I added the FhirConverterErrorCode as that is meant to give more granular nsights on when/what error caused this exception. So I feel that should lead us to the exact point.

Reply:
But I can also add that check if we feel the exception type is not enough to indicate that.